opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/dto/result/body/AccountListDetailBody.java

```diff
@@ -0,0 +1,24 @@
+package de.adorsys.opba.protocol.api.dto.result.body;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+
+@Builder
+@Data
+@Getter
+public class AccountListDetailBody {
```

Comment:
Probably this class should reside in `opba-facade-protocol-api-shared`

Reply:
done